bin/ui5.cjs

```diff
@@ -98,8 +111,8 @@ module.exports = ui5;
 
 if (process.env.NODE_ENV !== "test" || process.env.UI5_CLI_TEST_BIN_RUN_MAIN !== "false") {
 	ui5.main().catch((err) => {
-		console.log("Fatal Error: Unable to initialize UI5 CLI");
-		console.log(err);
+		process.stderr.write("Fatal Error: Unable to initialize UI5 CLI");
+		process.stderr.write(err);
```

Comment:
`TypeError [ERR_INVALID_ARG_TYPE]: The "chunk" argument must be of type string or an instance of Buffer or Uint8Array. Received an instance of Error`

Reply:
It was due to the introduced event listeners. Now it's removed. Please check the comments bellow: 
- https://github.com/SAP/ui5-cli/pull/686#issuecomment-1966144261
- https://github.com/SAP/ui5-cli/pull/686#issuecomment-1966154218